using/self_service.md

```diff
@@ -322,3 +322,118 @@ Once allocation has been created, you can see its access information:
 2. Account ID that needs to be passed to *sbatch* command when scheduling a job.
 
 ![Details of an allocation](../images/project-batch-details.png)
+
+## Adding a public SSH key to a profile
+1. Generate your SSH keypair (on Windows: using PuTTYgen, Linux: using OpenSSH)
+2. Click Details on the left side of the screen
+3. Select SSH keys
+4. Click Add SSH key
+5. Paste there contents of a public SSH key
+
+![Adding public SSH key](../images/adding-ssh-key.png)
```

Comment:
This screenshot contains invalid caption. Please remove this caption from screenshot.

See also:

https://english.stackexchange.com/questions/59640/what-is-the-difference-between-dialogue-and-dialog

Reply:
why